Plugins/CartonBundlePlugin/CartonBundlePluginCommand.swift

```diff
@@ -85,12 +85,12 @@ struct CartonBundlePluginCommand: CommandPlugin {
         ["--resources", $0.string]
       } + extractor.remainingArguments
     let frontend = try makeCartonFrontendProcess(context: context, arguments: frontendArguments)
-    frontend.forwardTerminationSignals()
-    try frontend.run()
-    frontend.waitUntilExit()
-    if frontend.terminationStatus == 0 {
-      print("Bundle written in \(bundleDirectory)")
-    }
-    frontend.checkNonZeroExit()
+    try frontend.checkRun(
+      printsLoadingMessage: false,
+      didExit: {
+        print("Bundle written in \(bundleDirectory)")
```

Comment:
It seems `didExit` is called regardless the process succeeded or failed, so the message is printed even though the frontend process failed.

Do we really need `didExit` hook point? It might be unnecessary if we move the message printing to the frontend

Reply:
挙動を変えてしまっていますね、すみません。
何も考えずにロジックを維持しようとしましたが、たしかに先のコマンドが自分でメッセージを出せば良さそうです。やってみます。